app/src/tingapp.js

```diff
@@ -14,7 +14,13 @@ class Tingapp {
 
         const newDocumentPath = path.join(tempDir, 'untitled.tingapp');
         console.log(newDocumentPath);
-        fsextra.copySync('./default.tingapp', newDocumentPath);
+        // check for dev to select where to fetch resources from
+        if(/[\\/]electron-prebuilt[\\/]/.test(process.execPath)){
+          fsextra.copySync('./app/default.tingapp', newDocumentPath);
+        }else{
+          fsextra.copySync(path.join(process.resourcesPath,'app.asar','default.tingapp'), newDocumentPath);
+        }
```

Comment:
I did a bit of research around this, as it looks brittle. Would [`process.defaultApp`](http://electron.atom.io/docs/api/process/#processdefaultapp) work instead? Added in electron v1.0.2.